.github/bin/s3/setup-empty-s3-bucket.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+S3_SCRIPTS_DIR="${BASH_SOURCE%/*}"
+
+S3_BUCKET_IDENTIFIER=trino-s3fs-ci-$(openssl rand -hex 8)
+
+S3_BUCKET_TTL=$(date -u -d "+2 hours" +"%Y-%m-%dT%H:%M:%SZ" 2>/dev/null || date -u -v "+2H" +"%Y-%m-%dT%H:%M:%SZ")
+
+echo "Creating an empty AWS S3 bucket ${S3_BUCKET_IDENTIFIER} in the region ${AWS_REGION}"
+S3_CREATE_BUCKET_OUTPUT=$(aws s3api create-bucket \
+  --bucket "${S3_BUCKET_IDENTIFIER}" \
+  --region "${AWS_REGION}")
+
+if [ -z "${S3_CREATE_BUCKET_OUTPUT}" ]; then
+    echo "${S3_CREATE_BUCKET_OUTPUT}"
```

Comment:
This is always empty in this case, maybe add some error message?

Reply:
added error msg